capstone/src/main/webapp/statistics_script.js

```diff
@@ -92,17 +136,81 @@ function loadBubbleChart(listA, listB, listC, listD) {
   chart.draw(stats, options);
 }
 
-function loadPieSentimentChart(list) {
+function loadCompletionPieChart(obj) {
```

Comment:
It may be helpful for readers if you choose a more descriptive argument name instead of `obj` and `list` :-)

Reply:
I've made the changes :)